standard_plus_issue/security/res_groups.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
```

Comment:
@ventilooo Groups usually should use `noupdate=1` to allow post-install adjustments.

Reply:
done